### PR TITLE
docs: add contributors map json file

### PR DESCRIPTION
**Pull request by actuallyyun**
Issue #893

Is `/assets` a good place for this file? 
For the map UI, I assume we want to put it in `/about` page, what do you think? @vnugent

### CODE REVIEW

**Comment by vnugent** [comment]
What do you think if we use Geojson format instead of regular json?  The benefit is GitHub natively supports geojson file. 

Example taken from your data

```json
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "firstName": "Yun",
        "favoriteCrag": "8fe4fa78-fda9-5742-b807-00b2c31d7bcd"
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          0.905152180850496,
          41.26613040056221
        ]
      },
      "id": 0
    }
  ]
}
```


```geojson
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "firstName": "Yun",
        "favoriteCrag": "8fe4fa78-fda9-5742-b807-00b2c31d7bcd"
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          0.905152180850496,
          41.26613040056221
        ]
      },
      "id": 0
    }
  ]
}
```

**Comment by vnugent** [comment]
> Issue #893
> 
> Is `/assets` a good place for this file? For the map UI, I assume we want to put it in `/about` page, what do you think? @vnugent

Yes

File: src/assets/contributors-map.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "first_name": "yun",
+    "last_name": "ji",
+    "github_id": "actuallyyun",
+    "home_coordinates": "41.26613040056221, 0.905152180850496",
```
Comment (vnugent) [inline src/assets/contributors-map.json]: FYI for privacy reason you don't have to share your exact home coordinates, a city landmark, a nearby crag is good enough.

Comment (actuallyyun) [reply src/assets/contributors-map.json]: Good point! I will put a note in the instructions.

**Comment by actuallyyun** [comment]
> ```json
> "id": 0
> ```

That's nice. How should we handle the `"id"` field? Ask people to manually increment?

**Comment by vnugent** [comment]
> > ```json
> > "id": 0
> > ```
> 
> That's nice. How should we handle the `"id"` field? Ask people to manually increment?

`id` is not required.  We can take that field out.

Can you also mention the order of coordinates is  [longitude, latitude]?

**Comment by actuallyyun** [comment]
> > > ```json
> > > "id": 0
> > > ```
> > 
> > 
> > That's nice. How should we handle the `"id"` field? Ask people to manually increment?
> 
> `id` is not required. We can take that field out.
> 
> Can you also mention the order of coordinates is [longitude, latitude]?

Will do!

**Review by vnugent — APPROVED** [review]

**Pull request merged**